google/ggrc-core
# GGRC-3671 Controls for sorting are not centered in tree view

# Issue description

Arrows are not centered in tree view header.

# Steps to test the changes

Steps to reproduce:
1. My work page > Assessments tab
2. Look at the controls for sorting (grey triangles)
*Actual Result:* Controls for sorting are not centered in tree view
*Expected Result:* Controls for sorting should be centered in tree view.

# Solution description

The solution is to use existing sortable-column component in tree view.

# Sanity checklist




# PR Review checklist
